Public/Src/Cache/ContentStore/Distributed/NuCache/BlobCentralStorage.cs

```diff
@@ -42,22 +42,29 @@ public class BlobCentralStorage : CentralStorage
         /// <nodoc />
         public BlobCentralStorage(BlobCentralStoreConfiguration configuration)
         {
-            Contract.Requires(configuration.ConnectionStrings.Count != 0);
+            Contract.Requires(configuration.Credentials.Count != 0);

```

Comment:
>Credentials [](start = 44, length = 11)

I realize you are not chaing things here, but I'm checking that we somwhere have a friendly user message that there must be at least one credential int he configuration, since this is a hard crash which will likely not result in something actionalbe for end user...

Reply:
Added requires where they should be and added a message to them. Is that good enough, or should I use a different mechanism?

---
In reply to: [293122690](https://github.com/microsoft/BuildXL/pull/459#discussion_r293122690) [](ancestors = 293122690)